tests/installation/disk_activation_zfcp.pm

```diff
@@ -0,0 +1,73 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: This test module handles ZFCP disk activation
+#          through libyui-rest-client.
+
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use YuiRestClient::Logger;
+
+sub run {
+    # in Disk Activation page press "ZFCP" button
+    my $disk_activation_selector = $testapi::distri->get_disk_activation_selector();
+    $disk_activation_selector->zfcp();
+
+    # in the ZFCP Device overview press the "Add" button
+    my $zfcp_configuration_overview = $testapi::distri->get_zfcp_configuration_overview();
+    $zfcp_configuration_overview->add();
+
+    # In the Add Device page insert hard wired channel id and press next
+    my $zfcp_add_disk = $testapi::distri->get_zfcp_add_disk();
+    $zfcp_add_disk->set_channel("0.0.fa00");
+    $zfcp_add_disk->next();
+
+    # yui.log from 1st VR showed that it takes a while for
+    # device detection so lets sleep a bit.
+    sleep(8);
```

Comment:
Please, never use sleep in a test, I cannot find right now the document where this is explicitly forbidden, or ok, not recommended by contributors of this repo. In general is a very bad practice in any language, because you never know exactly how much to wait. We use it though for our framework because we need some sort of timeout but not in a test.

Also I don't thing you need it, the next page should just check with its `exists` method and this arch is not slow.